src/main/java/ca/bc/gov/chefs/etl/util/JsonUtil.java

```diff
@@ -133,6 +133,16 @@ public static String fixPcnName(String payload) {
         return result;
     }
 
+    public static String fixPcnNameObject(String payload) {
+        // The following code aims to replace occurrences of "pcnName":"{}" with "pcnName": "", as "pcnName" is
+        // expected to be a String and not an object
+        String pcnPattern = "\"pcnName\":\\{\\}";
+        String pcnReplacement = "\"pcnName\": \"\"";
+
+        String result = payload.replaceAll(pcnPattern, pcnReplacement);
+        return result;
```

Comment:
I know this is the same as other existing methods but there's no point in defining result only to return it.